### PR TITLE
Fix serving static content in sphinx docs

### LINKED ISSUE #29: Make the automatic docs on thoth-station.github.io work with with jekyll
**Describe the bug**

As we generate part of the web page with jekyll (automatically on each puth to https://github.com/thoth-station/thoth-station.github.io), the documentation generated using sphinx stopped working.

One of the issues that were spotted was CSS/JS placement that was ignored based on Jekyll logic. Unfortunatelly `include` directive in jekyll _config.yaml did not work on GitHub (locally it worked). The fix to address this was introduced in https://github.com/thoth-station/nepthys/commit/56b63dd0c6bfb4ed4922d5fadbcc00466cbdb9ef.

Another issue comes with images and modules that are placed into `_images` and `_modules` directory. As all the files prefixed with underscore are ignored by Jekyll, these static files do not show up in the documentation.

**To Reproduce**
1. Go to https://thoth-station.ninja/docs/developers/adviser/introduction.html
2. See no images rendered

1. Go to https://thoth-station.ninja/docs/developers/adviser/thoth.adviser.html#module-thoth.adviser.exceptions
2. Click on "source" for any Python object
2. See 404 reported

**Expected behavior**

Images and module source code should be rendered and available.

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/nepthys/issues/29

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: create_docs.sh [hunk 1]
```diff
@@ -6,17 +6,17 @@ die() { echo "$*" 1>&2 ; exit 1; }
...
 do
-    # git clone --depth 1 https://github.com/thoth-station/${repo}.git clones/${repo}
+    git clone --depth 1 https://github.com/thoth-station/${repo}.git clones/${repo}
     # Copy _templates to each repo for Google analytics functionality.
     if  [[ $GITHUB_COMMIT = "1" ]]; then
         cp -r _templates/ clones/${repo}/docs/source/
     fi
     pushd clones/${repo}
-    # pipenv install
+    pipenv install
```
Comment (goern) [inline create_docs.sh:19]: no micropipenv here?

**Comment by goern** [comment]
can we double check with what operate-first is doing wrt docs? @harshad16 @durandom

File: create_docs.sh (hunk 1 above)
Comment (fridex) [reply create_docs.sh:19]: Pipenv is already used during docs generation for creating env and installing deps. We can switch to micropipenv, but I would do it in a separate PR then.

**Comment by fridex** [comment]
> can we double check with what operate-first is doing wrt docs? @harshad16 @durandom

Can this be merged meanwhile? The docs generation is broken without this. As the cronjob updates docs periodically, it breaks the docs on each push.

**Review by harshad16 — APPROVED** [review]
lgtm,
thanks :100:

**Pull request merged**